docs/source/graph_preprocessor.rst

```diff
@@ -0,0 +1,41 @@
+Graph-Based Feature Extraction
+==============================
+
+The GraphFeaturePreprocessor is a scikit-learn compatible preprocessing library that enables feature extraction from graph-structured data.  
+The library includes utilities for creating, updating, and transforming graph-structured data. 
+The basic input and output data representation used by the library is an edge list with edge features.
```

Comment:
I would add a screenshot with an example of such data (a table with a few transactions and columns and column names. It's ok to say edge list with edge features, but not sure all people understand this terminology/format.

Reply:
ok, ignore this comment please, just saw you added it a table below, great.